diff --git a/template.php b/template.php
index 08886d3..3691d90 100644
--- a/template.php
+++ b/template.php
@@ -267,10 +267,14 @@ function zurb_foundation_preprocess_block(&$variables) {
 
   // Add classes based on region.
   switch ($variables['elements']['#block']->region) {
-    // Add a striping class
+    // Clear blocks in this region
     case 'sidebar_first':
+      $variables['classes_array'][] = 'clearfix';
+    break;
+    // Add a striping class & clear blocks in this region
     case 'sidebar_second':
       $variables['classes_array'][] = 'block-' . $variables['zebra'];
+      $variables['classes_array'][] = 'clearfix';
     break;
 
     case 'header':
